client/components/codeEditor.js

```diff
@@ -34,10 +34,27 @@ const _onChange = event => {
   dispatchEvent('codeEditor:change');
 };
 
+let MacCMDPressed = false;
+
+const MacCommandKeys = ['MetaLeft', 'MetaRight'];
```

Comment:
what does this do?

Reply:
So for mac if you want to run the code using the `Command` key you cannot use the normal `event.ctrlKey + enter` to run the command.

The key to the left and right `Command` keys on MAC are `MetaLeft/Right` so if you pressed the `Command` key it will set `MacCMDPressed` to true and if your next keystroke is `enter` it will run the command.


On any key that is not `MetaLeft/Right` it will set `MacCMDPressed` back to false